script.js

```diff
@@ -6,6 +6,75 @@ const mainBarContent = document.querySelector('.main-bar');
 const mainContent = document.querySelector('.main-content');
 const MainSection = document.querySelector('.main-main');
 
+const closeModal = document.querySelector('#close-modal');
+const seeProject = document.querySelectorAll('.see-project-btn');
+const containerModal = document.querySelector('.container-modal');
+const modalTitle = document.querySelector('.modal-title h3');
+const modalDescription = document.querySelector('.bottom-modal-desktop p');
+const modalDescriptionMob = document.querySelector('.bottom-modal-desc p');
+const modalImage = document.querySelector('.moda-image-all');
+
+const seeProjectMobile = document.querySelectorAll('.modal-mobile-see-pro');
+
+const project = {
+  0: {
+    title: 'Tonic',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: './images/project_1.png',
+  },
+  1: {
+    title: 'Multi-Post Stories',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: './images/project_image2.png',
+  },
+  2: {
+    title: 'Tonic',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: './images/project_image3.png',
+  },
+  3: {
+    title: 'Multi-Post Stories',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: './images/project_image4.png',
+  },
+};
```

Comment:
Nice work implementing the objects👍👍 but I suggest this should be an array of objects so you can easily loop through it and render it dynamically😉😉 also kindly add the `technologies used` and `link to live version` for each objects as these are also part of the contents of the popup and project section😉😉😉